sprovflow-viewer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM tomcat
```

Comment:
Edited this docker file such that it builds form local checked out files instead of downloading from a repository. In this way the docker file and the code version are always in sync.